install/infra/single-cluster/gcp/terraform.tfvars

```diff
@@ -1,8 +1,8 @@
 # the cluster_name should be of length less than 15 characters and surrounded by double quotes
-cluster_name =
+cluster_name = "gitpod"
 
 # a cloudDNS zone and certificate request will be created for this domain; surround the domain name within double quotes
-domain_name =
+domain_name = "your_domain_name.com"
```

Comment:
**Opinion:** it's a very little known fact, but [`example.com`](https://www.iana.org/domains/reserved) is a reserved domain for demonstration examples, as is the [`.example` TLD](https://en.wikipedia.org/wiki/.example). Something to have in our back pocket for cases like this.